makefile

```diff
@@ -1,4 +1,4 @@
-CC ?= gcc
+CC ?= cc
```

Comment:
Can you provide a reason on this, I don't have a problem with it but why?

Reply:
I think that `cc` is a link to the default compiler usually.

Reply:
Yes, it does, just tested a while ago and CC is a symbolic link to GCC `cc -> gcc` 
Similar to how the sh command works.

I guess I don't have any issues with this change, feel free to resolve this when you read it.

Reply:
`cc` is the standard name for your C compiler. What happens, when you don't have `gcc` installed?

Reply:
I don't really want to uninstall gcc, however I'd assume if possible, clang would take its place else there would be one.

Reply:
And that's why it should be `cc`. Virtually all OSes agree that the C compiler is symlinked to `cc`. On Darwin, `clang` is symlinked to `gcc`, but that isn't the case everywhere. Just why would you depend on the C compiler used, if you don't have to?